nominees/os-city.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "OS City",
+  "aliases": [
+    "Onesmart"
+  ],
+  "description": "An improved, self-sovereign digital ID to make life easy.",
+  "website": "https://os.city",
+  "license": [
+    {
+      "spdx": "AGPL-3.0",
```

Comment:
Minor note: This and the following 2 repositories are GPL-3.0, not AGPL-3.0

Reply:
Fixed in commit d9f20ed